### PR TITLE
Update CreateApi to 0.2.0

### LINKED ISSUE #248: Update CreateAPI to version 0.2.0
I've recently started looking at the options offered by CreateApi to customize the code generated for my own project, but the currently used version, 0.0.5, still uses the old option naming and doesn't support some newer options, so I've decided to try to update to 0.2.0. Before opening a PR, here are the changes that would happen to the existing code.

- The header comment lines `// swiftlint:disable all` are removed. If they are still wanted, the `fileHeaderComment` option could be extended to include it again. See https://github.com/CreateAPI/CreateAPI/pull/103
- operation ids are always added and `.get`, `.post` and `.patch` are replaced by `Request(method: "GET", url: path, body: body)` etc. E.g.
```
 public func get(fieldsSandboxTesters: [FieldsSandboxTesters]? = nil, limit: Int? = nil) -> Request<AppStoreConnect_Swift_SDK.SandboxTestersV2Response> {
    .get(path, query: makeGetQuery(fieldsSandboxTesters, limit))
 }
```
would become
```
 public func get(fieldsSandboxTesters: [FieldsSandboxTesters]? = nil, limit: Int? = nil) -> Request<AppStoreConnect_Swift_SDK.SandboxTestersV2Response> {
    Request(path: path, method: "GET", query: makeGetQuery(fieldsSandboxTesters, limit), id: "sandboxTestersV2-get_collection")
 }
```
See https://github.com/CreateAPI/CreateAPI/pull/102 and https://github.com/CreateAPI/CreateAPI/pull/83

Can I go on with a PR?

### PR DESCRIPTION
**Pull request by nickasd**
Closes #248.

### CODE REVIEW

File: Sources/OpenAPI/Generated/Entities/Platform.swift [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-//
-// swiftlint:disable all
 
 import Foundation
 
 public enum Platform: String, Codable, CaseIterable {
 	case ios = "IOS"
 	case macOs = "MAC_OS"
 	case tvOs = "TV_OS"
-    case visionOs = "VISION_OS" // VisionOS manually added since version 3.0 of Apple's OpenAPI spec fails to include it. Reported in FB13208526 27/9/2023.
```
Comment (mathiasemil) [inline Sources/OpenAPI/Generated/Entities/Platform.swift:12]: FYI, we need to manually keep this line until Apple fixes `VISION_OS` missing in the spec or we come up with a better solution to handle this.

Comment (nickasd) [reply Sources/OpenAPI/Generated/Entities/Platform.swift]: Thanks, just added it.

**Comment by AvdLee** [comment]
@nickasd due to the last merge, there are some merge conflicts to solve. Could you have a look? 🙏

**Comment by nickasd** [comment]
> @nickasd due to the last merge, there are some merge conflicts to solve. Could you have a look? 🙏

Done, although there was no need to merge the OpenAPI 3.0 update, since mine is also updating to OpenAPI 3.1.

**Comment by nickasd** [comment]
The PR seems to be stale now. Do I still need to do anything?

**Comment by AvdLee** [comment]
@nickasd definitely not! Merging in. Sorry, I've been a bit busy with other work lately! Let's goooo!

And thanks a lot for your contribution again! 🙏 Highly appreciated

**Review by AvdLee — APPROVED** [review]

**Pull request merged**